app/views/measures/_measure.html.erb

```diff
@@ -51,11 +51,15 @@
   <% end %>
 
   <td class="conditions-col govuk-table__cell">
-    <% if measure.has_measure_conditions? %>
-      <%= link_to "Conditions",  "#", class: 'reference', 'data-popup-ref' => "#{measure.destination}-#{measure.id}-conditions", "aria-label": "Click to open a dialog with the conditions for this measure", role: "button" %>
-    <% elsif show_proofs_for_geographical_areas?(roo_schemes, measure) %>
-      <%= link_to "Conditions",  "#", class: 'reference', 'data-popup-ref' => "#{measure.destination}-#{measure.id}-cds-proofs", "aria-label": "Click to open a dialog with the conditions for this measure", role: "button" %>
-    <% end %>
+    <div data-controller="anchor">
+      <% if measure.has_measure_conditions? %>
+        <%= link_to "Conditions", "#", class: 'reference', data: { action: "click->anchor#launchModal", modal_ref: "#{measure.destination}-#{measure.id}-conditions" }, "aria-label": "Click to open a dialog with the conditions for this measure", role: "button" %>
+        <div id="modal" class="modal tariff-info" data-controller="modal" data-anchor-target="modal"></div>
+      <% elsif show_proofs_for_geographical_areas?(roo_schemes, measure) %>
+        <%= link_to "Conditions", "#", class: 'reference', data: { action: "click->anchor#launchModal", modal_ref: "#{measure.destination}-#{measure.id}-cds-proofs" }, "aria-label": "Click to open a dialog with the conditions for this measure", role: "button" %>
+        <div id="modal" class="modal tariff-info" data-controller="modal" data-anchor-target="modal"></div>
```

Comment:
same modal div here

Reply:
This modal div must sit within the anchor div which cannot move outside the scope of the row cell.

Reply:
see previous comment